exchanges/order/timeinforce_test.go

```diff
@@ -9,6 +9,27 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/encoding/json"
 )
 
+func TestTimeInForceIs(t *testing.T) {
+	t.Parallel()
+	tifValuesMap := map[TimeInForce][]TimeInForce{
+		GoodTillCancel | PostOnly:   {GoodTillCancel, PostOnly},
+		GoodTillCancel:              {GoodTillCancel},
+		GoodTillCrossing | PostOnly: {GoodTillCrossing, PostOnly},
+		GoodTillDay:                 {GoodTillDay},
+		GoodTillTime:                {GoodTillTime},
+		GoodTillTime | PostOnly:     {GoodTillTime, PostOnly},
+		ImmediateOrCancel:           {ImmediateOrCancel},
+		FillOrKill:                  {FillOrKill},
+		PostOnly:                    {PostOnly},
+		GoodTillCrossing:            {GoodTillCrossing},
+	}
+	for tif := range tifValuesMap {
+		for _, v := range tifValuesMap[tif] {
```

Comment:
🦍 Might be missing something, but why not use the value from the map in the first place?
```suggestion
	for tif, exps := range tifValuesMap {
		for _, v := range exps {
```